### DISCUSSION TITLE
When will Large Hosted Runners Have macOS Support?

### DISCUSSION THREAD

**Discussion by gblikas** [community/community#40401] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

I'm currently in the beta-program for Large Github Runners. There are some great options for windows and ubuntu, but macOS is still stuck at a 3 core option. Is there an expected release date, or a larger (unlisted) runner option for macOS?

**Comment by ossanna16** [comment]
Hi there @gblikas and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by fhammerl** [comment, marked as answer]
More powerful macOS runners are planned to come in Q1 2023

As the issue says, these runners are not part of the 'Larger Runners' ecosystem, rather you'll be able to pick them by editing your `runs-on` in your workflow YAML.


Public roadmap item:
https://github.com/github/roadmap/issues/506

**Reply by StephenHodgson** [reply]
This is great, but it doesn't provide the names of the runners in question
